src/main/java/TreeHandlers/MainTreeHandler.java

```diff
@@ -177,9 +185,11 @@ public State handleInput(int input){
                                 CornerTilePerformActionUseCase cornerTileInteractor = new CornerTilePerformActionInteractor();
                                 CornerTiles cornerTile = (CornerTiles) landedOnCell;
                                 answer = cornerTileInteractor.performAction(currentPlayer,board, cornerTile);
-
+                                break;
                             case "Action Space":
-                                // TODO add action space case
+                                PerformActionSpaceUseCase actionSpaceInteractor = new PerformActionSpaceCardInteractor();
+                                ActionSpace2 actionSpace2 = (ActionSpace2) landedOnCell;
```

Comment:
My review references this line specifically.

Reply:
well i was thinking more just deleting ActionSpace and renaming ActionSpace2, so that IntelliJ just does it for you.